hek/handler.py

```diff
@@ -355,43 +354,35 @@ def _get_tag_hash(self, data, def_id, data_key, hashes, is_meta,
 
     def get_tagref_invalid(self, parent, attr_index):
         '''
-        Returns whether or not the filepath of the tag reference isnt valid.
-        Returns False if the filepath is empty or the file exists.
-        Returns True otherwise.
+        Checks if filepath of a tag reference is invalid.
+        Returns False if file exists.
+        Returns True if does not or if the reference is empty.

```

Comment:
This part of the docstring is wrong. It returns false if the path is empty, not true.

Reply:
nice catch. this might've been me changing my mind about how this function should work at some point, and never updating the docstring.